src/shipit_static_analysis/shipit_static_analysis/config.yml

```diff
@@ -2,19 +2,36 @@ target: obj-x86_64-pc-linux-gnu
 
 # All the clang checks used by the bot
 clang_checkers:
- - -*
- - clang-analyzer-deadcode.DeadStores
- - modernize-loop-convert
- - modernize-raw-string-literal
 # modernize-use-auto (controversial, see bug 1371052)
 # modernize-use-bool-literals (too noisy because of `while (0)` in many macros)
- - modernize-use-equals-default
- - modernize-use-equals-delete
- - modernize-use-nullptr
- - modernize-use-override
- - mozilla-*
- - performance-faster-string-find
- - performance-for-range-copy
- - readability-else-after-return
- - readability-misleading-indentation
- - readability-redundant-control-flow
+ - name: -*
+   publish: no
+ - name: clang-analyzer-deadcode.DeadStores
+   publish: no
+ - name: modernize-loop-convert
+   publish: yes
+ - name: modernize-raw-string-literal
+   publish: yes
+ - name: modernize-use-equals-default
+   publish: yes
+ - name: modernize-use-equals-delete
+   publish: yes
+ - name: modernize-use-nullptr
+   publish: yes
+ - name: modernize-use-override
+   publish: no
+ - name: mozilla-*
+   publish: no
+ - name: performance-faster-string-find
+   publish: no
+ - name: performance-for-range-copy
+   publish: yes
```

Comment:
Could you please replace the two `performance-` rules by a global

- [ ] performance-*

?